lib/plugins/aws/lib/getErrorAssistance.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+/* eslint max-len: ["error", 120] */
+
+const invalidCronSyntax = (error) => {
+  let message = '';
+  const resourceProperties = JSON.parse(error.ResourceProperties);
+
+  message += `You have invalid syntax in a "schedule" event. You used "${resourceProperties.ScheduleExpression}".`;
+  message += '\n';
+  message += 'You can use "rate" syntax (e.g.: rate(2 hours)) or "cron" syntax (e.g.: cron(0 12 * * ? *)).';
+  message += '\n';
+  message += 'Click http://amzn.to/2yiWTc5 for AWS documentation on valid syntax.';
+
+  return message;
+};
+
+const bucketNotEmpty = (error) => {
+  let message = '';
+
+  message += 'Your deployment S3 bucket still has objects in it, so it could not be removed.';
+  message += '\n';
```

Comment:
This should be more generic as it isn't always the deployment bucket. Something along the lines of "An S3 bucket in this stack still has objects in it..."